### ISSUE TITLE
Cannot give in amounts

### ISSUE THREAD

**Issue by namthip53** [BlueWallet/BlueWallet#3243]
Hi, 

As European Users you cannot give in decimal amounts like for example 0.5 Btc or 0.6 Euros/dollars. The pop up keyboard only gives numbers and , (comma) NOT . (dot) as selection. There is no way around to get the . (dot) selected on the given keyboard.

Note: After starting up Bluewallet it is possible to use , (comma) 1 time. After that you have to close Bluewallet and start again to get another try.

Solution would be that , (comma) = .(dot) in Bluewallet and set , (comma)  to . (dot) automatically when used.

Use iPhone X IOS 14.6

Thx!

## Are you reporting a bug?

Please provide:

* your phone model and OS version
* BlueWallet app version (settings->about->scroll down)
* self-test passes? Open settings->about->scroll down, tap "Run self-test"
* unique ID for our crash reporting service (settings->about->scroll down, tap "copy")

## Proposing a feature?

Go right ahead
